src/Horarium.IntegrationTest/IntegrationTestBase.cs

```diff
@@ -1,14 +1,36 @@
 using System;
+using Horarium.Mongo;
+using Horarium.Repository;
 
 namespace Horarium.IntegrationTest
 {
     public class IntegrationTestBase
     {
         protected const string IntegrationTestCollection = "IntegrationTestCollection";
-        protected const string DatabaseNameMongo = "IntegrationTestScheduler";
+        private static readonly string DatabaseNameMongo = "IntegrationTestScheduler" + Guid.NewGuid().ToString();
+
+        private readonly string _connectionMongo =
+            $"mongodb://{Environment.GetEnvironmentVariable("MONGO_ADDRESS") ?? "localhost"}:27017/{DatabaseNameMongo}";
         
-        protected readonly string ConnectionMongo =
-            $"mongodb://{Environment.GetEnvironmentVariable("MONGO_ADDRESS")??"localhost"}:27017/{DatabaseNameMongo}";
+        protected HorariumServer CreateHorariumServer()
+        {
+            var dataBase = Environment.GetEnvironmentVariable("DataBase");
```

Comment:
How do we run tests from IDE with this change? 
Probably should leave a note about this line, like 
> Select database manually changing this line in case of running test in IDE 

or smth like than

Reply:
<img width="1088" alt="image" src="https://user-images.githubusercontent.com/11731874/59367794-824f3680-8d45-11e9-9d45-74cab3f93bc7.png">
I think intergration test will be work localy in docker compose with full enviroment.